### PR TITLE
docs: convert to modern doc system

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

**Fixes:** #21

## Summary:

Convert old react style docs to new format used in wc-generator.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [x] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [ ] My update follows the CONTRIBUTING guidelines of this project
- [ ] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by jason-capsule42** [comment]
@blackfalcon this is one of the two draft PRs that needed your input regarding how to integrate custom function calls and style imports for demo page example code blocks. Please let me know if you need additional information.

**Comment by blackfalcon** [comment]
This is looking pretty good so far. You can delete `index_old.html` when you are done transferring the content. Also, please build the other `.html` files. Dialog and Drawer.

File: demo/index.html [hunk 1]
```diff
@@ -1,331 +1,51 @@
...
+        });
+    </script>
+    <script type="module" src="../src/component-base.js"></script>
+    <script type="module" src="../src/auro-dialog.js"></script>
+    <script type="module" src="../src/auro-drawer.js"></script>
+    <script src="https://unpkg.com/@alaskaairux/auro-accordion@latest/dist/auro-accordion__bundled.js" type="module"></script>
+    <script src="https://unpkg.com/@alaskaairux/auro-button@latest/dist/auro-button__bundled.js" type="module"></script>
+    <script src="https://unpkg.com/@alaskaairux/auro-header@latest/dist/auro-header__bundled.js" type="module"></script>
+
+    <script>
```
Comment (blackfalcon) [inline demo/index.html:38]: These functions should be abstracted into a single file that can be imported into the scope of the demos here and also be referenced in the demo page for the docsite.

File: demo/index_old.html [hunk 2]
```diff
@@ -0,0 +1,331 @@
+<!DOCTYPE html>
```
Comment (blackfalcon) [inline demo/index_old.html:1]: This file can be deleted once the new demo pages have been created.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Pull request closed without merging**